setup.py

```diff
@@ -44,7 +44,7 @@ def initialize_options(self):
     def get_git_describe(self):
         cmd = ["git", "describe"]
         process = Popen(cmd, stdout=PIPE)
-        output = process.communicate()[0].strip()
+        output = process.communicate()[0].strip().decode('utf-8')
```

Comment:
It returns bytes (see https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate). We *could* set `universal_newlines=True` in the Popen command instead but, meh.